app/templates/people/person/positions/minister/edit.hbs

```diff
@@ -0,0 +1,260 @@
+{{page-title "Bewerk"}}
+
+<div class="au-c-body-container au-c-body-container--scroll">
+    <div class="au-o-box au-o-flow au-o-flow--large">
+        <PageHeader>
+        <:title>
+            Positie:
+            {{@model.minister.ministerPosition.function.label}}
+            <br />
+            {{@model.minister.ministerPosition.worshipService.classification.label}}
```

Comment:
I don't think the classification label needs to be shown here? It looks a bit weird I think. Did you see it in a design or was it mentioned in a discussion somewhere?

Reply:
The minister index page shows it, that's why I "inherited" the classification. But in fact none of the two pages has it in Figma. We should probably change both of them?